### ISSUE TITLE
CTC_Greedy_Decoder outputs sum of logits and not logprobs

### ISSUE THREAD

**Issue by Garvys** [tensorflow/tensorflow#8561]
## CTC_Greedy_Decoder outputs sum of logits and not logprobs

I have been writing my own CTC_Greedy_Decoder function in order to decode the output of a RNN trained with the CTC loss. To check my version, I decided to compare my outputs with TensorFlow's ones. We had the same first output (the decoded label) but we had big differences on second (the logprobs).

After some debugging, I figured out that TF's GreedyDecoder outputs the opposite of the sum of the higher logit of each column. Whereas, in the documentation it is written that we should expect logprobs. 

### How to fix:
- Change the documentation from logprobs to logits
- Instead of summing the logits, it would be  better to sum the higher logit of each column minus the logsumexp of the column. With that fix, the output will be logprobs as written in the documentation.

### Environment info
Operating System: Ubuntu 16.04
TensorFlow: 1.0.1 running on CPU

**Comment by drpngx** [comment]
Do you mind submitting a PR and CC @ebrevdo ?

**Comment by Garvys** [comment]
I can do that. Should I make a PR to change the doc or to change the code so that the function actually returns logprobs ?

**Comment by drpngx** [comment]
We use un-normalized logits everywhere, so I would go for changing the docs.

**Comment by ebrevdo** [comment]
+1

**Comment by Garvys** [comment]
PR : #8690

**Comment by gunan** [comment]
Looks like this is already fixed by #8690.
Closing issue.
